apiv2/serializers.py

```diff
@@ -46,23 +46,12 @@
 DEFAULT_FIELDS_IN_PACK_DETAIL = None  # Separated by commas (None = all)
 
 
-def get_sound_analyzers_output_helper(sound, fallback_to_db=True):
+def get_sound_analyzers_output_helper(sound):
     analyzers_output = {}
+    sound_analysis_dict = {an.analyzer: an.analysis_data for an in sound.analyses.all()}
     for analyzer_name, analyzer_info in settings.ANALYZERS_CONFIGURATION.items():
         if 'descriptors_map' in analyzer_info:
-            query_select_name = analyzer_name.replace('-', '_')
-            analysis_data = None
-            if hasattr(sound, query_select_name):
-                analysis_data = getattr(sound, query_select_name)
-                if type(analysis_data) == str:
-                    analysis_data = json.loads(analysis_data)
-            else:
-                if fallback_to_db:
-                    # Retrieve the analysis data from the db
-                    try:
-                        analysis_data = sound.analyses.get(analyzer=analyzer_name, analysis_status="OK").analysis_data
```

Comment:
If there are no items in `sound.analyses.all()` then we'll just get `None` from the dictionary. This is different to the previous bulk method which had an attr on the sound which could have been None. In this case it doesn't make sense to get the item from the database "again". 
If we didn't do a prefetch, then the above call to `sound.analyses.all()` will get them anyway.

There's a slight difference here where we we might get items with a non-OK status. We could fix this by modifying the dict comprehension